schema_poc/schema_registry.py

```diff
@@ -4,6 +4,7 @@
 
 RESPONSE_KEY_VERSION = "version"
 RESPONSE_KEY_SCHEMA = "schema"
+RESPONSE_KEY_SUBJECT = "subject"
```

Comment:
This isn't actually part of the response from RedPanda.  We should only be using these constants as part of what RedPanda sends us back when we make requests on it.